src/Geometry/GEllipse.class.st

```diff
@@ -133,6 +143,8 @@ GEllipse >> intersectionsWithLine: aGLine [
 	k := center y.
 	a := self semiMajorAxisLength.
 	b := self semiMinorAxisLength.
+	
+	(a = b and: [ a = 0 ]) ifTrue: [ ^ {center} asSet ].
```

Comment:
```suggestion
	(a = b and: [ a = 0 ]) ifTrue: [ ^ Set with: center ].
```

Reply:
Would it be possible to have a test indicating in which case this is needed? To avoid regression in the future